site/en/docs/crux/methodology/index.md

```diff
@@ -0,0 +1,567 @@
+---
+# Required
+layout: 'layouts/doc-post.njk'
+
+# Required
+title: CrUX Methodology
+
+# Required
+# This appears in the ToC of the project landing page at
+# /docs/[project-name]/. It also appears in the <meta description> used in
+# Google Search.
+description: >
+  Technical documentation on CrUX eligibility, metrics, dimensions and accessing the data.
+
+# Optional
+# This appears below the title and is an optional teaser
+subhead: >
+  This section documents how CrUX collects and organizes user experience data.
+
+# Required
+date: 2022-06-23
+
+# Optional
+# Include an updated date when you update your post
+# updated: 2020-10-16
+
+# Optional
+# How to add a new author
+# /docs/handbook/how-to/add-an-author/
+#authors:
+
+# Optional
+# How to a new tag
+# /docs/handbook/how-to/add-a-tag/
+tags:
+  - web-vitals
+---
+
+## Eligibility
+
+At the core of the CrUX dataset are individual user experiences, which are aggregated into page-level and origin-level distributions. This section documents user eligibility and the requirements for pages and origins to be included in the dataset. All eligibility criteria must be satisfied in order for an experience to be included in page-level data available in PageSpeed Insights and the CrUX API: [User](#user-eligibility), [Origin](#origin-eligibility) and [Page](#page-eligibility). Experiences which meet the User and Origin criteria (but not Page) will be included in the origin-level data available in all CrUX data sources.
+
+Pages and origins will be automatically included or removed from the dataset if their eligibility changes over time. There is not currently a way to manually submit pages or origins for inclusion.
+
+### User {: #user-eligibility}
+
+For a **user** to have their experiences aggregated in the CrUX dataset, they must meet the following criteria:
```

Comment:
I'm not native english speaker, is it right to say 'they must' for a user?

Reply:
This sounds ok to me